### PR TITLE
Add participation overview and acceptable conduct

### LINKED ISSUE #8: Code of conduct for Participation and Communication
We will use terms such as Student or Team Member, and use knowledge from our readings of what makes a good team member in software engineering, and have that back up our overview and our acceptable and unacceptable conduct. 

We will use mostly one line statements that end with a period.

### PR DESCRIPTION
**Pull request by LiamBlack3**
## Description

Added writing to participation overview and for acceptable participation conduct

### Type of Change

- [ ] Feature
- [ ] Bug fix
- [x] Documentation

## Contributors

- @LiamBlack3

## Reminder

All GitHub Actions checks must be in a passing state in this branch or fork before this pull request can be merged.

### CODE REVIEW

File: conduct.md [hunk 1]
```diff
@@ -68,13 +68,13 @@ TODO: Write an introductory paragraph.
 
 ### Participation
 
-TODO: Write an introductory paragraph.
+Participation for students will be a multidide of factors, we all have bad days, we all have topics we know more or less about, but it is being prepared each day having done the homework, coming to class with an open mind to learn and making an effort to be apart of either small group or large group conversations that will reflect students participation.
```
Comment (ullrichd21) [inline conduct.md]: The first sentence seems to be a bit hard to follow for me, I think the bit after `but it` is what's confusing me? Is there a way you can make that flow better? 

The first sentence is a run on as well I think.

File: conduct.md [hunk 2]
```diff
@@ -68,13 +68,13 @@ TODO: Write an introductory paragraph.
...
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students will respect all students opinions.
```
Comment (ullrichd21) [inline conduct.md]: Maybe `peers` instead of using students twice?

File: conduct.md [hunk 3]
```diff
@@ -68,13 +68,13 @@ TODO: Write an introductory paragraph.
...
+- Students will show up to every class ready to learn and engage with the class.
+- Students participation will be a combination of small group effort and/or large group discussion effort.
```
Comment (ullrichd21) [inline conduct.md:78]: Should we include something about filling out the participation surveys?

**Review by ullrichd21 — CHANGES_REQUESTED** [review]
Overall this looks good. I have included a couple minor changes I think could improve readability.

**Review by nathandloria — DISMISSED** [review]
Once Danny's suggestions are resolved, this looks good to merge!

**Review by ullrichd21 — APPROVED** [review]
Looks good now!

**Review by nathandloria — APPROVED** [review]
Yayyyyyy :)
